upgrade/roles/upgrade_idrac_telemetry/tasks/filter_idrac.yml

```diff
@@ -0,0 +1 @@
+../../../../telemetry/roles/idrac_telemetry/tasks/filter_idrac.yml
```

Comment:
Add return line at the EOF

Reply:
this is symbolic link. does not require EOF